src/views/Networking.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <Page class="league-page-container">
+    <div
+      class="networking-page"
+    >
+      <input
+        v-model="room"
+        :placeholder="$t('Networking.Placeholder')"
+      >
+      <router-link :to="'/league/' + room">
```

Comment:
It should be possible to write this way: 
```suggestion
      <router-link :to="{ name: 'conference', params: { room } }">
```
https://router.vuejs.org/api/#to

Reply:
I know. But it's not work this way. Not sure why. Also Erik ask me to rename route to `league`

Reply:
Sorry for misunderstanding. It was not work with `network/:room` route. But it's fine with `league` or `conference`

Reply:
@davidyuk no. It also don't work with
```js
<router-link :to="{ name: 'league', params: { room } }">
        {{ $t('Networking.Start') }}
      </router-link>
```

That's why I used concatenation.

Reply:
https://stackoverflow.com/a/44781258 🤷‍♀️